lib/scutil/host-fp-folding.c

```diff
@@ -81,13 +81,22 @@ configure_denormals(bool denorms_are_zeros, bool flush_to_zero)
   fenv_t fenv;
   if (fegetenv(&fenv) != 0)
     fprintf(stderr, "fegetenv() failed: %s\n", strerror(errno));
-#if defined(__x86_64__) && !defined(_WIN32)
-  fenv.__mxcsr &= ~0x0040;
-  if (denorms_are_zeros)
-    fenv.__mxcsr |= 0x0040;
-  fenv.__mxcsr &= ~0x8000;
-  if (flush_to_zero)
-    fenv.__mxcsr |= 0x8000;
+#ifdef __x86_64__
+  #ifdef _WIN32
+    fenv._Fe_stat &= ~0x0040;
+    if (denorms_are_zeros)
+      fenv._Fe_stat |= 0x0040;
+    fenv._Fe_stat &= ~0x8000;
+    if (flush_to_zero)
+      fenv._Fe_stat |= 0x8000;
+  #else
```

Comment:
Need to use `_controlfp` to set these correctly. I'm having trouble figuring out what the existing constants do, since they're from linux headers.

Reply:
Values:
```
0000 0000 0000 0000 0100 0000
0000 0000 1000 0000 0000 0000
```

Reply:
From the intel documentation:
- "Denormals are zeros"
- "Flush to zeros"

@isuruf Can you confirm this?

Reply:
Yes, those values correspond to DAZ and FTZ bits

Reply:
Also see https://softwareengineering.stackexchange.com/a/337251